release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2784,6 +2784,30 @@ mount: /var/lib/kubelet/plugins/kubernetes.io/local-volume/mounts/local-pv-bc42d
 To workaround the issue, delete the `cloud-event-proxy-store-storage-class-http-events` `PVC` CR and re-deploy the PTP Operator.
 (link:https://issues.redhat.com/browse/OCPBUGS-12358[*OCPBUGS-12358*])
 
+[id="ocp-4-13-ran-known-issues-post-ga"]
+* During {ztp-first} provisioning of a {sno} managed cluster with secure boot enabled in the `SiteConfig` CR, multiple `ProvisioningError` errors are reported for the `BareMetalHost` CR during host provisioning.
```

Comment:
@yliu127 Can you review/lgtm on this known issue text for https://issues.redhat.com/browse/OCPBUGS-8434 please.

Reply:
/lgtm